go/internal/initutil/metrics.go

```diff
@@ -0,0 +1,67 @@
+package initutil
+
+import (
+	"flag"
+	"net"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.uber.org/zap"
+)
+
+func (m *Manager) SetupMetricsFlags(fs *flag.FlagSet) {
+	fs.StringVar(&m.Metrics.Listener, "metrics.listener", ":8888", "Metrics listeners")
+	fs.StringVar(&m.Metrics.Handler, "metrics.handler", "/bty/metrics", "Metrics handler path")
+	fs.BoolVar(&m.Metrics.Pedantic, "metrics.pedantic", true, "Enable Metrics pedantic")
+}
+
+func (m *Manager) GetMetricsRegistery() (*prometheus.Registry, error) {
```

Comment:
```suggestion
func (m *Manager) GetMetricsRegistry() (*prometheus.Registry, error) {
```

there are other similar typos